Examples/Run-MATLAB-Tests.md

```diff
@@ -0,0 +1,65 @@
+# Run MATLAB Tests on Jenkins Server
```

Comment:
Can the `Examples` folder be changed to something like `doc`? Or at least `examples`, lowercase. That would align more with the folder naming convention of Maven projects.

Reply:
+1 on changing the folder name to examples

Reply:
I renamed it to examples and placed it under src. This is consistent with the MathWorks doc folder structure (matlab/doc).